packages/design-system-tokens/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "include": ["src/*.ts", "node_modules", "tests/*.ts"],
```

Comment:
There doesn't actually appear to be any `src` folder. We could make a src folder and put `index.ts`, `lib`, and `tokens`, into it. That would be more consistent with our other packages.